packages/react/src/components/table/Table.tsx

```diff
@@ -232,7 +237,8 @@ export const Table = ({
   dense = false,
```

Comment:
Not related to this exact fix but should we have a default value for dataTestId as well?

Reply:
- [ ] Add default value for dataTestId